birdhouse/default.env

```diff
@@ -1,5 +1,7 @@
 # All env in this default.env can be overridden by env.local.
-# All env in this default.env must not depend on any env in env.local.
+
+# All env in this default.env must NOT depend on any other env.  If they do,
+# must use single quote and must add to the list of DELAYED_EVAL.
```

Comment:
I think it would be worth adding the reason here that "[...] must use single quote *to avoid early expansion before overrides in local.env are applied* [...]".

Reply:
done in 407c348e576b917657bc8b2de1e3b5e5f8c45fe5